lib/eventum/class.notification.php

```diff
@@ -1019,7 +1019,7 @@ public function notifySubscribers($issue_id, $emails, $type, $data, $subject, $i
                     $full_subject = "[#$issue_id] $subject: $extra_subject";
                 }
             } elseif (($type == 'new_issue') && ($is_assigned)) {
-                $full_subject = "[#$issue_id] New Issue Assigned: " . $data['iss_summary'];
+                $full_subject = "[#$issue_id] " . ev_gettext('New Issue Assigned') . ": " . $data['iss_summary'];
```

Comment:
this applies few other places you've changed. plz review all your changes critically.